# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../panda/hotfix.patch
# Copyright (C) 2007 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./objects.c.orig	2007-03-06 11:09:57.000000000 -0300
+++ ./objects.c	2007-03-06 11:10:18.000000000 -0300
@@ -99,7 +99,7 @@
   created->children = (panda_child *) panda_xmalloc (sizeof (panda_child));
 
   ((panda_child *) created->children)->next = NULL;
-  (panda_child *) created->cachedLastChild = NULL;
+  created->cachedLastChild = NULL;
 
   // By default this object is not a pages object
   created->isPages = panda_false;
